packages/rule-tester/src/utils/config-validator.ts

```diff
@@ -25,8 +26,8 @@ const ruleValidators = new WeakMap<AnyRuleModule, ValidateFunction>();
 let validateSchema: ValidateFunction | undefined;
 const severityMap = {
   error: 2,
-  warn: 1,
   off: 0,
+  warn: 1,
```

Comment:
this is another case where the value ordering is better than the prop name ordering